definitions/v3/torrentseeds-api.yml

```diff
@@ -28,7 +28,6 @@ caps:
     tv-search: [q, season, ep, imdbid, tvdbid]
     movie-search: [q, imdbid, tmdbid]
     music-search: [q]
-    book-search: [q]
```

Comment:
Should we really be removing this?

Reply:
they don't have nor support books, so why would we support it?